documentation/modules/ROOT/partials/modules/all-connectors/proc-triggering-an-incremental-snapshot.adoc

```diff
@@ -0,0 +1,249 @@
+Currently, the only way to initiate an incremental snapshot is to send an xref:{link-signalling}#debezium-signaling-ad-hoc-snapshots[ad hoc snapshot signal] to the signaling {data-collection} on the source database.
+tag::sql-based-snapshot[]
+You submit a signal to the signaling {data-collection} as SQL `INSERT` queries.
+end::sql-based-snapshot[]
+tag::nosql-based-snapshot[]
+You submit a signal to the signaling {data-collection} by using the MongoDB `insert()` method.
+end::nosql-based-snapshot[]
+
+After {prodname} detects the change in the signaling {data-collection}, it reads the signal, and runs the requested snapshot operation.
+
+The query that you submit specifies the {data-collection}s to include in the snapshot, and, optionally, specifies the kind of snapshot operation.
+Currently, the only valid option for snapshots operations is the default value, `incremental`.
+
+To specify the {data-collection}s to include in the snapshot, provide a `data-collections` array that lists the {data-collection}s or an array of regular expressions used to match {data-collection}s, for example, +
+tag::sql-based-snapshot[]
+`{"data-collections": ["public.MyFirstTable", "public.MySecondTable"]}` +
+end::sql-based-snapshot[]
+tag::nosql-based-snapshot[]
+`{"data-collections": ["public.Collection1", "public.Collection2"]}` +
+end::nosql-based-snapshot[]
+
+The `data-collections` array for an incremental snapshot signal has no default value.
+If the `data-collections` array is empty, {prodname} detects that no action is required and does not perform a snapshot.
+
+[NOTE]
+====
+If the name of a {data-collection} that you want to include in a snapshot contains a dot (`.`) in the name of the database, schema, or table, to add the {data-collection} to the `data-collections` array, you must escape each part of the name in double quotes. +
+ +
+For example, to include a table that exists in the `*public*` schema and that has the name `*My.Table*`, use the following format: `*"public"."My.Table"*`.
```

Comment:
We may want to conditionalize even this with a NoSQL vs SQL bit just due to the wording used.

Reply:
Took me a few reads to understand which "this" you were referring to. The note, right? I'll make the necessary changes to that. thanks.